package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lob/ui-components",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
do you need to update package-lock.json too?

Reply:
🤦 Yep, sorry, I always forget to do that.